### DISCUSSION TITLE
Help with Modern PnP search

### DISCUSSION THREAD

**Discussion by JimEddAnt** [microsoft-search/pnp-modern-search#4132] [Q&A]
Hi Everyone, I'm trying to build a search that excludes the person searching's details from the results 

We've built a database that holds employee documentation and built a PnP Modern search for user's to find other employees with (i.e. managers looking at their Teams documents, etc) but we want to have the search in such a way that the documentation for the person searching doesn't show.

We've mapped a crawled property for the email address to a refinable string and updated the query to exclude it from the search using the property name and the alias but this didn't work.

Looking for any advice - thank you

**Comment by wobba** [comment]
Can you share how you have configured the web part KQL? Also take a look at https://microsoft-search.github.io/pnp-modern-search/usage/search-results/tokens/#user-tokens
